casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -422,35 +436,24 @@ object ProtoUtil {
   def stringToByteString(string: String): ByteString =
     ByteString.copyFrom(Base16.decode(string))
 
-  def basicDeployData(id: Int): DeployData = {
-    //TODO this should be removed once we assign the deploy with exact user
-    Thread.sleep(1)
-    val timestamp = System.currentTimeMillis()
-    val term      = s"@${id}!($id)"
-
-    DeployData()
-      .withUser(ByteString.EMPTY)
-      .withTimestamp(timestamp)
-      .withTerm(term)
-      .withPhloLimit(accounting.MAX_VALUE)
-  }
-
-  def basicDeploy(id: Int): Deploy = {
-    val d    = basicDeployData(id)
-    val term = InterpreterUtil.mkTerm(d.term).right.get
-    Deploy(
-      term = Some(term),
-      raw = Some(d)
+  def basicDeployData[F[_]: Monad: Time](id: Int): F[DeployData] =
```

Comment:
Hmm, I thought we were working to phase out usages of `shared.Time` in favour of `cats.effect.Clock`/`cats.effect.Timer`

Reply:
I'll let someone else handle all of them at once.

Reply:
@birchmd I think that with recent changes made by @sebtomba the `shared.Time` is now based on  the cats.